reference-test-app/components/VideoPlayerView/VideoPlayerView.brs

```diff
@@ -21,6 +21,13 @@ sub Init()
     }
     m.comscoreConnector.callFunc("configure", m.player, comscoreConfig)
 
+    m.aepConnector = m.top.findNode("THEOAEPConnector")
+    aepConfig = {configId: "<MY_CONFIG_ID>", domainName: "<MY_EDGE_DOMAIN>", mediaChannel: "My Channel", mediaPlayerName: "My Player", mediaAppVersion: "1.0", logLevel: 3 }
+    m.aepConnector.callFunc("configure", m.player, aepConfig)
+    m.aepConnector.observeField("ecid", "_onECIDChanged")
+    m.aepConnector.callFunc("getExperienceCloudId", _onECIDChanged, m)
+    m.aepConnector.callFunc("updateUserConsent", "y")
```

Comment:
For Comscore it is like `m.comscoreConnector.callFunc("updateUserConsent", "1")` 
Any specific reason why for AEP it has to be `"y"`?

Reply:
It has to do with the values used by the underlying libraries. Comscore does theirs with numbers and Adobe with letters. I think Adobe's makes more sense. Maybe I can make Comscore also handle letters and mark the numbers as deprecated. Then in the next major release we can switch Comscore wholly over to letters.

Reply:
@RameshDPrasad does that address your question?